pkg/inventory/inventory.go

```diff
@@ -96,10 +96,13 @@ func ValidateNoInventory(objs []*unstructured.Unstructured) error {
 	}
 }
 
-// splitUnstructureds takes a slice of unstructured.Unstructured objects and
-// splits it into one slice that contains the inventory object templates and
-// another one that contains the remaining resources.
-func SplitUnstructureds(objs []*unstructured.Unstructured) (*unstructured.Unstructured, []*unstructured.Unstructured, error) {
+// SplitUnstructureds takes of slice of Unstructured objects, and returns
+// the one inventory object (if it exists), and the slice of the remaining
+// objects. Returns nil as the inventory object (no error) if there is
+// not an inventory object in the passed slice. Returns an error if there
+// are multiple inventory objects in the passed slice.
+func SplitUnstructureds(objs []*unstructured.Unstructured) (
```

Comment:
This is a breaking change for the API. I think it'd be beneficial to make it a compilation failure so that anyone who updates the dependency will have to explicitly handle the situation. Otherwise it compiles but then doesn't work the way it used to.